integration-cli/docker_cli_search_test.go

```diff
@@ -89,3 +89,11 @@ func (s *DockerSuite) TestSearchCmdOptions(c *check.C) {
 		c.Fatalf("failed to search with stars&automated&no-trunc options on the central registry: %s", out)
 	}
 }
+
+// search for repos which start with "ubuntu-" on the central registry
+func (s *DockerSuite) TestSearchOnCentralRegistryWithDash(c *check.C) {
+	testRequires(c, Network, DaemonIsLinux)
+
+	out, exitCode := dockerCmd(c, "search", "ubuntu-")
+	c.Assert(exitCode, check.Equals, 0, check.Commentf("failed to search on the central registry: %s", out))
```

Comment:
post review nit, this whole line isn't needed, just `dockerCmd(...)` so if it fails it's failing the test and output will be printed in test :) no big deal tho